client/controllers/PlanningController.js

```diff
@@ -53,10 +53,12 @@ export function PlanningController(
                 store.dispatch(actions.resetStore())
             })
 
+            const App = $scope.vm.app || PlanningApp;
```

Comment:
Why do we need to have `PlanningApp` component on scope?

Reply:
this is just temporary so I can use same controller for both old and new app